ceph-extra-exporter/log.go

```diff
@@ -1,10 +1,32 @@
 package main
 
-import "github.com/cybozu-go/log"
+import (
+	"log/slog"
+	"os"
+)
 
-var logger *log.Logger
+var logger *slog.Logger
 
 func init() {
-	logger = log.NewLogger()
-	logger.SetFormatter(log.JSONFormat{})
+	utsname, err := os.Hostname()
+	if err != nil {
+		panic(err)
+	}
+
+	logger = slog.New(slog.NewJSONHandler(os.Stderr, &slog.HandlerOptions{
+		ReplaceAttr: func(groups []string, a slog.Attr) slog.Attr {
+			switch a.Key {
+			case slog.TimeKey:
+				a.Key = "logged_at"
+			case slog.LevelKey:
+				a.Key = "severity"
+			case slog.MessageKey:
+				a.Key = "message"
+			}
+			return a
+		},
+	})).With(
+		slog.String("topic", "ceph-extra-exporter"),
```

Comment:
I think this setting is redundant because there are no other topics in this code.
Because I don't know much about slog package, please let me know if this is some kind of the best practices.

Reply:
That's right. I'll fix it.